go/vt/vtgate/executor.go

```diff
@@ -284,24 +300,38 @@ func (e *Executor) handleExec(ctx context.Context, safeSession *SafeSession, sql
 	return qr, err
 }
 
-func (e *Executor) shardExec(ctx context.Context, safeSession *SafeSession, sql string, bindVars map[string]*querypb.BindVariable, target querypb.Target, logStats *LogStats) (*sqltypes.Result, error) {
+func (e *Executor) destinationExec(ctx context.Context, safeSession *SafeSession, sql string, bindVars map[string]*querypb.BindVariable, target querypb.Target, destination key.Destination, logStats *LogStats) (*sqltypes.Result, error) {
 	f := func() ([]*srvtopo.ResolvedShard, error) {
-		return e.resolver.resolver.ResolveDestination(ctx, target.Keyspace, target.TabletType, key.DestinationShard(target.Shard))
+		var result []*srvtopo.ResolvedShard
```

Comment:
Please name this variable rss, for consistency.

Also, can just be:
rss, err := ...

Reply:
Done.